tests/system/test/MEZOTransfers.test.ts

```diff
@@ -94,6 +94,68 @@ describe("MEZOTransfers", function () {
     });
   });
 
+  describe("multipleApproveAreHandledProperlyByTheCore", function () {
+    let receipt: any;
+    let tx: any;
+    let receipient = ethers.Wallet.createRandom().address;
+
+    before(async function () {
+      await fixture();
+    });
+
+    it("should allow approving BTC for the receipient", async function () {
```

Comment:
I would assert MEZO allowance is still 0 here.

Reply:
addressed in 1bba9e8a0b8d6a109a561f3f583958318a1c0ff7